### PR TITLE
Update sponsorship-prospectus.md

### PR DESCRIPTION
**Pull request by ronitblenz**

### CODE REVIEW

File: sponsorship-prospectus.md [hunk 1]
```diff
@@ -218,7 +235,7 @@ title Participants by profession (2024)
...
 | Price(USD)                                       | 10,000                                                                                                                                         | 6,000                                                            | 3,500                              | 1,600                                                | 500 - 800 |
-| Price(NPR)<sup>\*0</sup>                         | TBD                                                                                                                                            | TBD                                                              | TBD                                | TBD                                                  | TBD       |
+| Price(NPR)<sup>\*0</sup>                         | 3,50,000                                                                                                                                       | 2,50,000                                                         | 1,50,000                           | 1,00,000                                             | 80,000    |
```
Comment (sukso96100) [inline sponsorship-prospectus.md]: umm where did  you got the Nepalese rupees pricing?

Comment (ronitblenz) [reply sponsorship-prospectus.md]: Hey, I got them from Aaditya. We were on a call to discuss those. Also, do let us know if you need any changes there.

Comment (sukso96100) [reply sponsorship-prospectus.md]: understand :) could you leave this tbd for now? let's get NPR pricing updated after we got some more updates from Aaditya

Comment (sukso96100) [reply sponsorship-prospectus.md]: Could you update pricing for local sponsors (NPR) based on sponsorship team discussed on chat?

- Diamond 10,00,000
- Gold 600000
- Silver 400000
- Bronze 200000 (Booth add-on 30000) 
- Supporter 70000 - 100000

Comment (sukso96100) [inline sponsorship-prospectus.md]:
```suggestion
| Price(NPR)<sup>\*0</sup>                         | 10,00,000                                                                                                                                       | 600000                                                         | 400000                           | 200000                                             | 70000 - 100000    |
```

**Comment by ronitblenz** [comment]
Oh you already made the changes. I would have done it tonight. Thanks btw!

**Pull request merged**